### ISSUE TITLE
Role install instructions on Galaxy incorrect

### ISSUE THREAD

**Issue by geerlingguy** [William-Yeh/ansible-fluentd#17]
I have traditionally used `ansible-galaxy install williamyeh.fluentd` to install this role... and that still works. But if you go to the Galaxy role page for this role (https://galaxy.ansible.com/william-yeh/fluentd), it shows:

    ansible-galaxy install william-yeh.fluentd

...which does _not_ work. You have to still use williamyeh for the namespace. This may be an upstream issue with Galaxy.

**Comment by jurajseffer** [comment]
The old name has just stopped working.

**Comment by geerlingguy** [comment]
@jurajseffer it looks like an upstream bug that was caused by a migration; see https://github.com/ansible/galaxy/issues/775

Specifically, in the last comment:

> If you find more roles that you want renamed from '_' to '-', please open a new issue, so that we don't miss it.

So it looks like someone needs to open an issue in https://github.com/ansible/galaxy and mention this?

**Comment by geerlingguy** [comment]
Well... now I get:

```
$ ansible-galaxy install williamyeh.fluentd
- downloading role 'fluentd', owned by williamyeh
 [WARNING]: - williamyeh.fluentd was NOT installed successfully: - sorry, williamyeh.fluentd was not found on
https://galaxy.ansible.com.
```

So you have to use `william-yeh.fluentd` now.

**Comment by jurajseffer** [comment]
Yeah it's properly butchered. https://galaxy.ansible.com/williamyeh says 15 roles, https://galaxy.ansible.com/william-yeh says 4 roles.
